app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -1,10 +1,23 @@
-@import "./variables.less";
+// Primary Color
+@primaryColorDefault: #578dc9;
+@primaryColor: var(--allPagesPrimaryColor, @primaryColorDefault);
+@primaryColorLight: var(--allPagesPrimaryColorLight, lighten(@primaryColorDefault, 10%));
+
+// Other base colors
+@darkGrey: var(--allPagesDarkGrey, #4d5466);
+@greyColorLighten1Default: #a8b3c5;
+@greyColorLighten1: var(--allPagesGreyColorLighten1, @greyColorLighten1Default);
+@toolbarLightIconColor: var(--allPagesToolbarLightIconColor, @greyColorLighten1Default);
```

Comment:
This should be removed to use variable.less

Reply:
Done